Formula/molten-vk.rb

```diff
@@ -33,13 +34,13 @@ class MoltenVk < Formula
   resource "Vulkan-Headers" do
     # ExternalRevisions/Vulkan-Headers_repo_revision
     url "https://github.com/KhronosGroup/Vulkan-Headers.git",
-        revision: "3ef4c97fd6ea001d75a8e9da408ee473c180e456"
+        revision: "ec9b96aae53e152f6812e66fa43757c1256369a0"
   end
 
   resource "SPIRV-Cross" do
     # ExternalRevisions/SPIRV-Cross_repo_revision
```

Comment:
Did these resources not match the specified revisions for 1.1.10?

Reply:
They don't; I can separate SPIRV-Cross into `head do`/`stable do` sections if you'd prefer (I don't think it's necessary for `Vulkan-Headers`, since it's bumped by a single commit, which just adds the new Metal extensions).

Reply:
Can we just pull `Vulkan-Headers` and `SPIRV-Cross` from head when also doing `brew install --HEAD molten-vk`?

Head resources should either always match what's in stable (so this should be the existing revisions, unless they're wrong), or not require updating (so we pull from head).

Reply:
How do I do that?

Is there any way to grab resource revisions from the main package's source? They're basically doing something submodule-style here.

Reply:
See the `dmd` formula for an example where we use specific artefacts for `stable` but pull from head for `head`.

Reply:
How's this? Seems to work fine here, at least?